### DISCUSSION TITLE
Select behavior of ApplicationModules.verify() to only detect cycles, but not check for API or Internal packages

### DISCUSSION THREAD

**Discussion by nawabb** [spring-projects/spring-modulith#84] [Ideas]
As per https://docs.spring.io/spring-modulith/docs/0.1.0-M1/reference/html/#verification the verify() method does two things:
1. No cycles on the application module level
2. Efferent module access via API packages only
It will be nice to be able to specify, only verify 1 (that there are no cycles present) but it's okay even if some modules are accessed in internal package 

@odrotbohm let us know your thoughts.

**Comment by odrotbohm** [comment]
I guess if you only want to do that, a simple slice ArchUnit rule would be sufficient. Preventing access to internal components is actually one of the important features of Spring Modulith on top of pure cycle prevention. I guess it would be weird to be able to disable that.
